### PR TITLE
[NUI] Apply missing comments for Scene3D demo

### PR DESCRIPTION
**Pull request by hinohie**
1. Collect all private values in upward of source code.
2. Formatting C#
3. MilliSeconds --> Milliseconds
4. Add some comments

Signed-off-by: Eunki Hong <eunkiki.hong@samsung.com>

### CODE REVIEW

File: test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs [hunk 1]
```diff
@@ -31,13 +31,75 @@ class Scene3DSample : NUIApplication
...
-    const int cameraAnimationDurationMilliSeconds = 2000; // milliSeconds
+    Model mModel;
 
     Animation mModelRotateAnimation;
-    const int modelRotateAnimationDurationMilliSeconds = 10000; // milliSeconds
+    const int modelRotateAnimationDurationMilliseconds = 10000; // milliseconds
+
+    private bool mMutex = false; // Lock key event during some transition / Change informations
+
+    #region Model list define
```
Comment (Seoyeon2Kim) [inline test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs:41]: It's really good to separate the regions like this :)

File: test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs [hunk 2]
```diff
@@ -31,13 +31,75 @@ class Scene3DSample : NUIApplication
...
+
+    #region Image based light list define
+    private static readonly List<(string, string)> IBLUrlList = new List<(string, string)>
+    {
+        ("forest_diffuse_cubemap.png", "forest_specular_cubemap.png"),
+        ("papermill_E_diffuse-64.ktx", "papermill_pmrem.ktx"),
+        ("Irradiance.ktx", "Radiance.ktx"),
+    };
+    private int currentIBLIndex = 0;
+    private float IBLFactor = 0.7f;
```
Comment (Seoyeon2Kim) [inline test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs:60]: To define the specific number makes the code much cleaner! :+1:

File: test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs [hunk 3]
```diff
@@ -31,13 +31,75 @@ class Scene3DSample : NUIApplication
...
+    };
+    private static readonly List<CameraInfo> CameraInfoList = new List<CameraInfo>()
+    {
+        // +Z front and -Y up.
+        new CameraInfo(){
+            Position = new Vector3(0.0f, 0.0f, 5.55f),
+            // Basic camera           : -Z front and -Y up
+            // After YAxis 180 rotate : +Z front and -Y up
+            // Note : Default camera has YAxis 180.0f rotation even we didn't setup.
+            Orientation = new Rotation(new Radian(new Degree(180.0f)), Vector3.YAxis),
```
Comment (Seoyeon2Kim) [inline test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs:80]: Thank you for adding the descriptions! :100:

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM!
Thank you for the clean code!

**Comment by hinohie** [comment]
I think camera direction comment was not correct. Will fix soon

**Pull request merged**